packages/frontend/src/components/molecules/NavBar.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Link from 'next/link';
+import { FaBars } from 'react-icons/fa';
+
+let pages = [
+  { title: 'Home', path: '#' },
+  { title: 'Jobs', path: '#' },
+  { title: 'Events', path: '#' },
+  { title: 'Companies', path: '#' },
+  { title: 'About Us', path: '#' },
+  { title: 'Contact us', path: '#' },
+];
+interface NavBarProps {
+  theme: string;
+  closeBar: () => void;
+}
+function NavBarLayout({ theme, closeBar }: NavBarProps) {
+  const selected = 'Home';
```

Comment:
selected should be dynamic not static, add it to the props of the layout, and if it was not defined or empty, then highlight none